### PR TITLE
💥 (node) upgrade node from v10 to v14

### PR DESCRIPTION
**Pull request by metal-gogo**
## PEW June 2021 summary

https://docs.google.com/document/d/1zC0DCXFjEYBRd4AMtknwfGn6BxukmwFe1v7Bi-AZlIE/edit#heading=h.njjjnaklunq6

## Dependencies that could not be updated

- `node-sass` starting on version `6.0.0` requires Webpack 5. But @vue-cli does not support it.
- `sass-loader` starting on version `11.0.0` requires Webpack 5. But @vue-cli does not support it.
- `babel-jest` requires having the same version for `jest`. @vue-cli installs jest for us and they use version `24.9.0`.

## Most important dependencies upgrades:

- Changes on https://github.com/homeday-de/homeday-blocks/commit/6baa3b3fb7e4eae5ad43b22c044f183a4789efd9:
  - Bumped Node version to `12.22.1` from `10.22.0`.
  - Changed: `"babel-core": "^7.0.0-bridge.0"` ---> `"@babel/core": "^7.14.6"`.
  - Changed: `"babel-eslint": "10.0.3"` ---> `"@babel/eslint-parser": "^7.14.5"`.
  - Added : `"@babel/runtime": "^7.14.6"`
  - Updated flacky test on `HdGaleryMedia`.
- Changes from https://github.com/homeday-de/homeday-blocks/pull/743/commits/3dfb71f958bedf170407bd0c8aadf4f39f02a163 to https://github.com/homeday-de/homeday-blocks/pull/743/commits/0a4de85164e9b5da9730441c26fd8bbf24d5801e:
  - Bumped Node version to `14.17.0` from `12.22.1`.
  - Erased `package-lock.json` and reinstalled all dependencies.
  - `HdPager` Keyboard events changed from `ArrowUp`, `ArrowLeft`, `ArrowRight`, `ArrowDown` to `Up`, `Left`, `Right`, `Down`. [Reference](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#example)
  - Use await everytime `setProps` is used. [Reference](https://vue-test-utils.vuejs.org/api/wrapper-array/setprops.html#setprops)
- Changes on https://github.com/homeday-de/homeday-blocks/pull/743/commits/b74b3b0bb5b8d7721b45955490866a9c79cb07f5
  - Bumped `husky` to version `6.0.0`
  - Removed `lint-staged` dependency.
- Changes on https://github.com/homeday-de/homeday-blocks/pull/743/commits/c79ed3d3715c28b3763acb7f8a831298cfb59976:
  - Bumped `storybook` to the latest version. 
- Changes on https://github.com/homeday-de/homeday-blocks/pull/743/commits/a24cf7dec8d2155d12b1dcce938ebcdeb42860c5:
  - The `vue-inline-svg` library included the `title` prop, so the behavior of the `HdIcon` was updated accordingly.

### CODE REVIEW

File: tests/unit/components/form/HdCheckbox.spec.js [hunk 1]
```diff
@@ -68,12 +68,12 @@ describe('HdCheckbox', () => {
...
 
-    wrapper.setProps({
+    await wrapper.setProps({
       value: true,
     });
-    checkboxInner.trigger('blur');
+    await checkboxInner.trigger('blur');
 
-    await wrapper.vm.$nextTick();
+    // await wrapper.vm.$nextTick();
```
Comment (kuroski) [inline tests/unit/components/form/HdCheckbox.spec.js:76]: 💅

**Review by kuroski — APPROVED** [review]
That was awesome work, my friend!
Thanks for upgrading the dependencies, fixing the testing inconsistencies, and also configuring husky

I just found one dead comment in one test, the rest it seems to be perfect

**Review by leandroinacio — APPROVED** [review]
Nice!

File: src/components/HdIcon.vue [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <inline-svg
     :src="src"
     :id="id"
+    :title="title"
```
Comment (ilyasmez) [inline src/components/HdIcon.vue:5]: Nice! 👍

**Review by ilyasmez — APPROVED** [review]
Great work @metal-gogo and @gepser! 🚀 👏 👏 👏
I've dropped just one tiny little suggestion, but other than that everything looks great!
And I also must admit that I didn't give the PR the review time it deserves, because it covers a big scope.

**Pull request merged**